### ISSUE TITLE
intégration schéma stationnement PMR dans schéma "lieux de stationnements"

### ISSUE THREAD

**Issue by virzinie** [cnigfr/schema-accessibilite-voirie#51]
bonjour 

dans le schéma "lieux de stationnement"
https://[schema.data.gouv.fr](https://schema.data.gouv.fr/etalab/schema-stationnement/0.1.5/documentation.html)/etalab/schema-stationnement/0.1.5/documentation.html
seul le nombre de places PMR est demandé mais sans lien avec description des places.
Comment pourrait-on lier les deux ? est-ce que des schémas et des données pivôt existent déjà ?

merci de vos retours

**Comment by GT-CNIG-DDU** [comment]
_Note préalable :_ 
Le schéma "[lieux de stationnement](https://github.com/etalab/schema-stationnement)" permet de modéliser les **stationnements en parcs** publics et privés à usage public, alors que le [standard CNIG Accessibilité](https://cnig.gouv.fr/ressources-accessibilite-a25335.html) décrit les places **de stationnement de véhicule sur voirie**, réservée aux personnes à mobilité réduite.
Cela ne correspond donc pas forcément aux mêmes places de stationnement...

> Est-ce qu'un schema existe déjà ?

Le standard CNIG Accessibilité [est référencé par schema.data.gouv](https://schema.data.gouv.fr/cnigfr/schema-accessibilite-voirie/) dans le cadre du catalogage des standard du CNIG, mais il ne fait pas (ou pas encore) l'objet d'un schema suivant le modèle TableSchema mis en œuvre par [schema.data.gouv](https://schema.data.gouv.fr/) ;

> Des données pivot existent déjà ?

Chaque objet de la classe `STATIONNEMENT_PMR` du standard CNIG porte un identifiant (attribut `idstationnement`).
Sa forme, compatible à la norme NeTEx, est décrite au [§4.3 du standard](https://cnig.gouv.fr/IMG/pdf/240215_standard_cnig_accessibilite_v2021-10_rev2024-02.pdf#page=41).

**Comment by virzinie** [comment]
Merci 
je reformule ma demande.
L'objectif serait de faire converger les deux pour décrire plus finement les places de stationnement dans les parkings en ouvrage, donc pour relier les deux schémas. Existe-t-il un schéma permettant de décrire les places PMR existantes dans les parkings en ouvrage (parcs). 
L'id de la classe stationnement_PMR est relatif à un stationnement en voirie, pour l'instant il n'y a pas d'id sur des places PMR dans les parcs de stationnement. 
à moins que ce ne soit pas utile de lier les deux schémas...

**Comment by GT-CNIG-DDU** [comment]
> Existe-t-il un schéma permettant de décrire les places PMR existantes dans les parkings en ouvrage (parcs) ?

Oui, certainement via la [norme NeTEx](https://normes.transport.data.gouv.fr/normes/netex/) et ses différents profils, dont : 
- **[NeTEx - Profil France - Parkings](https://normes.transport.data.gouv.fr/normes/netex/parkings/)**
- [NeTEx - Profil France - Accessibilité](https://normes.transport.data.gouv.fr/normes/netex/accessibilit%C3%A9/)
- [NeTEx - Profil France - Eléments communs](https://normes.transport.data.gouv.fr/normes/netex/elements_communs/)

> pour l'instant il n'y a pas d'id sur des places PMR dans les parcs de stationnement.

Je suppose que [NeTEx - Profil France - Parkings](https://normes.transport.data.gouv.fr/normes/netex/parkings/) définit un identifiant pour les places des parcs de stationnement.

Mais pour plus de précisions sur ces deux questions : poke @TuThoThai experte nationale NeTEx.

**Comment by TuThoThai** [comment]
Je confirme, un peu tardivemnt, que @GT-CNIG-DDU a tout à fait raison de pointer vers le profil Parking de NeTEx qui permet de décrire finement les places des parkings en ouvrage. La section en question est [Composants des Parkings](https://normes.transport.data.gouv.fr/normes/netex/parkings/#composants-des-parkings), notamment y est bien mentionné pour l'attribut `bays `
> Note : la description détaillée des places de parking est nécessaire en particulier dans le contexte des place PMR.
